OpenUpSA/wazimap-ng-ui
# Added line scrubber component to mapchip

## Description

Line scrubber component to mapchip

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-407

## How is it tested automatically?
Added a gui test

## How should a reviewer test it locally
enable linear scrubber from admin panel for a profile indicator
select that PI in FE 
Linear scrubber should be visible on mapchip when any subindicator from that PI is selected

## Screenshots
<img width="816" alt="Screenshot 2022-08-04 at 11 24 18 AM" src="https://user-images.githubusercontent.com/6871866/182773195-fceef44c-7164-4224-ae29-bfda20740ca6.png">


## Changelog

### Added
Added a new react component
Added a new trigger event

### Updated
failing unit test to not use onSubindicatorChangr

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
